### ISSUE TITLE
Json filtering doesn't work

### ISSUE THREAD

**Issue by nikolasburk** [prisma/prisma#2563]
## Bug description

Json filtering doesn't work in Prisma Client `findMany`.

## How to reproduce

Create a DB table in PostgreSQL like this:

```sql
CREATE TABLE myproject$dev."User" (
    id character varying(25) PRIMARY KEY,
    "jsonData" json,
);
```

Add the following JSON data into the column: `{"isRockstar":"true"}`

Introspect DB:

```prisma
model User {
  id        String    @id
  jsonData  Json?
}
```

Generate Prisma Client and run this query:

```ts
const users = await prisma.user.findMany({
  where: {
    jsonData: '{"isRockstar":"true"}',
  },
})
```

## Expected behavior

I expect this to return the `User` record with the respective data.

**Comment by pantharshit00** [comment]
I can confirm this. What is most likely happening here is prisma is trying to convert true into a boolean when we pass json as string. It works if I use a js object which is we convert to json instead:

```js
  const users = await client.user.findMany({
    where: {
      jsonData: '{"isRockstar":"true"}',
    },
  });
```
and it works:
![image](https://user-images.githubusercontent.com/22195362/83040479-ea815000-a05c-11ea-96cf-a97785d1a95e.png)


while 
```js
  const users = await client.user.findMany({
    where: {
      jsonData: '{"isRockstar":"true"}',
    },
  });
```

doesn't work:
![image](https://user-images.githubusercontent.com/22195362/83040357-cc1b5480-a05c-11ea-8714-2ee3c0a5a19c.png)

**Comment by pantharshit00** [comment]
Query sent to the engine:
```graphql
query {
  findManyUser(where: { jsonData: "\"{\\\"isRockstar\\\":\\\"true\\\"}\"" }) {
    id
    jsonData
  }
}
```

**Comment by do4gr** [comment]
Can you please check whether this still occurs and close it otherwise @pantharshit00 ?

**Comment by pantharshit00** [comment]
This is fixed in the latest dev version ` 2.6.0-dev.43`


The original example here was passing the json data as a string which is not valid. I think we changed the typings to disallow. This query works though which the the equivalent of the original query:
```ts
  const users = await prisma.user.findMany({
    where: {
      jsonData: { equals: { isRockstar: "true" } },
    },
  });

``
